### PR TITLE
Modified phonetic alphabet for lizards

### LINKED ISSUE #71967: Lizards cannot use radio uplink passwords containing the letter S
## Round ID:
196170 

## Reproduction:

Spawn as a lizard with radio uplink that has a password which contains the letter S. Proceed to be unable to open your uplink by stating the password.

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request
Fixes https://github.com/tgstation/tgstation/issues/71967 (lol)
Equal opportunity traitoring for lizards
## Changelog
:cl: LT3
fix: Lizardsss can now pronounce all the codesss for uplinksss
/:cl:

### CODE REVIEW

**Comment by Bawhoppen** [comment]
Hmmm I don't think this is the correct way to fix it...

**Comment by san7890** [comment]
i think you're doing a lot more harm than good by completely removing the phonetic equivalent for "s" and "x". would be wiser to perhaps filter certain words out in the uplink code

File: strings/phonetic_alphabet.txt [hunk 1]
```diff
@@ -12,15 +12,13 @@ Kilo
...
 Papa
 Quebec
 Romeo
-Sierra
 Tango
 Uniform
 Victor
-Whiskey
-X-ray
+Waterloo
```
Comment (Zonespace27) [inline strings/phonetic_alphabet.txt:22]: this isn't how the phonetic alphabet works

**Comment by lessthnthree** [comment]
> i think you're doing a lot more harm than good by completely removing the phonetic equivalent for "s" and "x". would be wiser to perhaps filter certain words out in the uplink code

Fair, I didn't think of anything else that may be using it.

**Comment by lessthnthree** [comment]
I'll open a new one when it's ready.

**Pull request closed without merging**